src/docs/src/config/replicator.rst

```diff
@@ -166,17 +166,39 @@ Replicator Database Configuration
 
         .. _inet: http://www.erlang.org/doc/man/inet.html#setopts-2
 
-     .. config:option:: valid_endpoint_protocols :: Replicator endpoint protocols
+    .. config:option:: valid_endpoint_protocols :: Replicator endpoint protocols
 
-        .. versionadded:: 3.3
+       .. versionadded:: 3.3
 
-        Valid replication endpoint protocols. Replication jobs with endpoint
-        urls not in this list will fail to run::
+       Valid replication endpoint protocols. Replication jobs with endpoint
+       urls not in this list will fail to run::
 
-            [replicator]
-            valid_endpoint_protocols = http,https
+           [replicator]
+           valid_endpoint_protocols = http,https
+
+    .. config:option:: valid_endpoint_protocols_log :: Log security issues with endpoints
+
+       .. versionadded:: 3.4
+
+       When enabled, CouchDB will log any replication that uses the insecure http
+       protocol::
 
-     .. config:option:: valid_proxy_protocols :: Replicator proxy protocols
+           [replicator]
+           valid_endpoint_protocols_log = true
+
+    .. config:option:: verify_ssl_certificates_log :: Log security issues with endpoints
+
+       .. versionadded:: 3.4
+
+       When enabled, and if ``ssl_trusted_certificates_file`` is configured
+       but ``verify_ssl_certificates`` is not, CouchDB will check the
+       validity of the TLS certificates of all sources and targets (
+       without causing the replication to fail) and log any issues::
+
+           [replicator]
+           verify_ssl_certificates_log = true
+
+    .. config:option:: valid_proxy_protocols :: Replicator proxy protocols
```

Comment:
We should probably add these to the default.ini commented with their respective defaults;

```
[replicator]

; Description....
;verify_ssl_certificates_log = true

```